File: C++/First Steps in Coding - Basics of C++/Conditional Statements - Advanced/Cinema Ticket.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;
int main() {
	string day;
	cin >> day;
	if (day == "Monday" or day=="Tuesday" or day=="Friday") {
		cout << "12";
	}
	else if (day == "Wednesday" or day=="Thursday") {
		cout << "14";
	}
	else if (day=="Saturday" or day=="Sunday") {
		cout << "16";
	}
}

File: Java/Advanced Conditional Statements/Lab/08. Cinema Ticket/CinemaTicket.java
import java.util.Scanner;

public class CinemaTicket {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String day = input.nextLine();
        int ticketPrice = 0;

        switch(day){
            case "Monday":
                ticketPrice = 12;
                break;
            case "Tuesday":
                ticketPrice = 12;
                break;
            case "Wednesday":
                ticketPrice = 14;
                break;
            case "Thursday":
                ticketPrice = 14;
                break;
            case "Friday":
                ticketPrice = 12;
                break;
            case "Saturday":
                ticketPrice = 16;
                break;
            case "Sunday":
                ticketPrice = 16;
                break;
        }

        System.out.println(ticketPrice);
    }
}
